tsconfig.json

```diff
@@ -12,5 +12,13 @@
 		// "noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
 		// "noFallthroughCasesInSwitch": true, /* Report errors for fallthrough cases in switch statement. */
 		// "noUnusedParameters": true,  /* Report errors on unused parameters. */
-	}
+		"resolveJsonModule": true,
```

Comment:
What's going on here 🤔

Reply:
Required in order to read the JSON Schema into the test. Stupid TypeScript rules.